### ISSUE TITLE
Trying to write an Exporter

### ISSUE THREAD

**Issue by Shinobubu** [arcanedreams/ArcManagedFBX#4]
I love the work you have done its very comprehensive.  I recently ran into a problem where I needed to recover some 3D assets that no longer have its originals so I thought hey I know the data format of these resource files maybe i should just rebuild the scene as an FBX.  So right now i'm trying your amazing FBX wrapper. But this project is not complete as I hoped it would be. 

I have ran into a problem where I can't create many of the necessary nodes to build a scene.
Things like FbxNode.Create() and FbxScene.Create()  wouldn't allow me to pass it the FBXManager. 
Many things missing for building a basic scene like FbxSurfacePhong and no ability to addMaterials to the nodes.  Essentially I only know how to build and export unpopulated empty scenes.  Export class works though.  I'm using the ExportDocument sample that came with the FBX 1015.1 SDK as a guide. I'm using VC 2012 with update 4.   I'm not very experienced with C/C++ so I'm at a loss.  Should I just give up if this is unattainable?

**Comment by LoveDuckie** [comment]
Apologies for taking so long to respond to this. It's been on my mind all week but I have been busy with life.

Unfortunately, as you have noticed, the library does not wrap the entirety of the FBX SDK. When I was developing the library I only made sure to wrap as much as I needed from it. After all, I wanted to make use of the library for my own purposes but later decided to open-source it in-case anyone else wanted to use or learn from it.

If you are not that familiar with either C++ or C++/CLI then I would recommend not using it for the time being. If you want me to notify you when I start work on it again, then send me an e-mail at [luc.shelton@arcanedreams.com](mailto:luc.shelton@arcanedreams.com) and I will let you know.

**Comment by Shinobubu** [comment]
Don't worry about it. Life takes precedence over my little pet project!  C++/CLI is very confusing hence I'm stepping back from this. Autodesk FBX is a big project. They changed there naming and namespaces in a matter of 2 versions.
I wrote a simple wrapper 4 years ago for importing meshes too but when the time came to compile it in a windows environment all of a sudden nothing worked. (Turns out the fbx dll doesn't like Ming at all)  And I have absolutely forgotten how I got it to work on my Mac and OpenSuse  <yikes> so I went to yours and others to replace my silly attempt at swig wrapped managed api.  This is really a giant land mine for someone who's has no formal programming education and training. I literally learned C/C++/C# via Google for christ sake. It seems like I only got lucky on my first attempts and now I'm way out of my depth.
Thanks for replying your work is still valuable :+1:
